Automattic/wp-calypso
# Persist scroll position in the Help Center

Fixes https://github.com/Automattic/dotcom-forge/issues/10075

## Proposed Changes

Persist the scroll position of the article element in memory so minimized the Help Center doesn't destory reading progress.

## Why are these changes being made?
To make reading articles more continuous.

## Testing Instructions

1. Open an article in the HC.
2. Scroll down.
3. Minimize the HC.
4. Maximize it.
5. The article should stay in place.
---
1. Open an article.
2. Scroll down.
3. Go back.
4. Go to another article. 
5. Should be at the top.
---
1. Open an article.
2. Scroll down.
3. Close the Help Center. 
4. Reopen it.
5. Go to an article.
6. Should be at the top.
---
1. Open the article "Getting Started on WordPress.com".
2. Scroll down to "Learn more about the dashboard".
3. Click it.
4. Scroll down in the new article.
3. Minimize the HC.
4. Maximize it.
5. The article should stay in place.
6. Click Back to go back to the original article.
7. Scroll should be preserved.

## Demo

https://github.com/user-attachments/assets/14cb22a2-64ea-4d09-8005-95a2241ead21
